### ISSUE TITLE
Unit tests failure while following the guide 

### ISSUE THREAD

**Issue by raikrahul** [Seagate/cortx-motr#219]
I am following the guide at, 

`doc/Quick-Start-Guide.rst`

While  I try to run the Unit tests, I see a possible assert failure 

```
 tx_bulk-empty                                   1.43 sec   64 MiB
  tx_bulk-error_reg  motr[08257]:   570  FATAL  [lib/assert.c:50:m0_panic]  panic: (!done) at m0_be_tx_bulk_put() (be/tx_bulk.c:618)  [git: sage-base-1.0-107-g780ec01] /var/motr/m0ut/m0trace.8257
Motr panic: (!done) at m0_be_tx_bulk_put() be/tx_bulk.c:618 (errno: 4) (last failed: none) [git: sage-base-1.0-107-g780ec01] pid: 8257  /var/motr/m0ut/m0trace.8257
```

The relevant output and system info are 

 ```
cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

```
 cat /proc/modules | grep lnet
lnet 586401 2 ksocklnd, Live 0xffffffffc06ad000 (OE)
libcfs 415252 2 ksocklnd,lnet, Live 0xffffffffc062e000 (OE)
```

```
$ uname -r
3.10.0-1062.18.1.el7.x86_64
```

**Comment by huanghua78** [comment]
Thank you for reporting.
This is known issue, and fix will be ready soon.

**Comment by huanghua78** [comment]
This issue has not been fixed.

**Comment by nezzzu** [comment]
@huanghua78 Any idea when this will be fixed? Please share with me if you have a plan. Also, is there a way to skip these failing tests and continue with the rest?

**Comment by huanghua78** [comment]
```
m0 run-ut --help
m0 run-ut -x be-ut:tx_bulk-error_reg
```

**Comment by nezzzu** [comment]
`  tx_bulk-empty                                   1.04 sec   64 MiB
  tx_bulk-error_payload  motr[30583]:  6010  FATAL  [lib/assert.c:50:m0_panic]  panic: (!done) at m0_be_tx_bulk_put() (be/tx_bulk.c:603)  [git: sage-base-1.0-133-gfa45ae7] /var/motr/m0ut/m0trace.30583
Motr panic: (!done) at m0_be_tx_bulk_put() be/tx_bulk.c:603 (errno: 4) (last failed: none) [git: sage-base-1.0-133-gfa45ae7] pid: 30583  /var/motr/m0ut/m0trace.30583
/home/seagate/github/cortx-motr/motr/.libs/libmotr.so.1(m0_arch_backtrace+0x20)[0x7ff068024920]
/home/seagate/github/cortx-motr/motr/.libs/libmotr.so.1(m0_arch_panic+0xe6)[0x7ff068024ad6]
/home/seagate/github/cortx-motr/motr/.libs/libmotr.so.1(+0x357ba4)[0x7ff068013ba4]
/home/seagate/github/cortx-motr/motr/.libs/libmotr.so.1(m0_be_tx_bulk_put+0x12f)[0x7ff067f76d4f]
/home/seagate/github/cortx-motr/ut/.libs/libmotr-ut.so.0(+0x131b64)[0x7ff0695f3b64]
/home/seagate/github/cortx-motr/ut/.libs/libmotr-ut.so.0(+0x131f8e)[0x7ff0695f3f8e]
/home/seagate/github/cortx-motr/ut/.libs/libmotr-ut.so.0(+0x1324de)[0x7ff0695f44de]
/home/seagate/github/cortx-motr/ut/.libs/libmotr-ut.so.0(m0_be_ut_tx_bulk_error_payload+0x4a)[0x7ff0695f48aa]
/home/seagate/github/cortx-motr/ut/.libs/libmotr-ut.so.0(m0_ut_run+0x25e)[0x7ff06973348e]
/home/seagate/github/cortx-motr/ut/.libs/lt-m0ut(main+0x1156)[0x4044b6]
/lib64/libc.so.6(__libc_start_main+0xf5)[0x7ff0667ba555]
/home/seagate/github/cortx-motr/ut/.libs/lt-m0ut[0x40460a]
/home/seagate/github/cortx-motr/utils/m0run: line 416: 30583 Aborted                 (core dumped) $(srcdir_path_of $binary) "$@"
`
Now I get this. Is this also part of the same issue?

**Comment by huanghua78** [comment]
Yes, I think so.

**Comment by nezzzu** [comment]
Now:
```
>>./scripts/m0 run-ut -x be-ut:tx_bulk-error_reg,be-ut:tx_bulk-error_payload
```
get me this far:
```
 stats                                           5.49 sec  270 MiB
  pool-sns-repair  motr[14497]:  eb40  FATAL  [lib/assert.c:50:m0_panic]  panic: m0_bob_check(&pump_bob, __amb) at cm_cp_pump_fom_tick() (cm/pump.c:386) *MAGIX(bt, bob) == bt->bt_magix [git: sage-base-1.0-133-gfa45ae7] /var/motr/m0ut/m0trace.14497
Motr panic: m0_bob_check(&pump_bob, __amb) at cm_cp_pump_fom_tick() cm/pump.c:386 (errno: 0) (last failed: *MAGIX(bt, bob) == bt->bt_magix) [git: sage-base-1.0-133-gfa45ae7] pid: 14497  /var/motr/m0ut/m0trace.14497
```
Looks like there are many. Do we have a list?

**Comment by huanghua78** [comment]
>pool-sns-repair

I did not see this.
"be" related failure is probably caused by some recent commit.
But this one is not seen recently by me or on Jenkins.

**Comment by nezzzu** [comment]
Will this be fixed as part of this issue? Or do we need to create a new issue?

**Comment by huanghua78** [comment]
This is fixed by 1c49493ca9e89c572cfd2cd50ccf99f8619ebd22
